src/components/PollSection/usePoll.ts

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useMemo, useState } from 'react';
+import { getPollById } from '../../providers/Social/pool.sdk';
+
+export const usePoll = (pollId: string, shouldFetch: boolean) => {
+  const [pollData, setPollData] = useState<Amity.Poll | undefined>(undefined);
+
+  const endDays = useMemo(() => {
+    if (pollData?.closedAt) {
+      const endDate = new Date(pollData.closedAt);
+      const differenceMs = endDate.getTime() - Date.now();
+      const daysRemaining = Math.ceil(differenceMs / (1000 * 60 * 60 * 24));
+      return daysRemaining || 0;
+    }
```

Comment:
imo, it would be better to handle this with `dayjs` or any date library

Reply:
Yeah. My thought is we are making library so we should use less  3rd party libraries as mush as possible to reduce  the chance of dependencies conflict with user's project. 

What do you think? It is your call.

Reply:
we already have `moment` as a dep already and for this case, we should use date lib because there are too many edge cases for dating thing

anyway, we should remove `add`, `yarn` from deps and change from `moment` to `dayjs`

Reply:
Updated !, removed moment and use dayjs